constants/__init__.py

```diff
@@ -75,6 +75,7 @@
             "torch_dtype": torch.bfloat16,
             "attn_implementation": "flash_attention_2",
         },
+        eval_block_delay=7200,  # ~1 day.
```

Comment:
Since we only sample the latest data from 18, should we make this something like 3-5x the time we expect each vali to produce a new sample? I suspect 4 hours would be plenty

Reply:
I also adjusted CORTEX_MAX_AGE. Let me know if you just wanted to change this one.